core/templates/components/question-directives/question-editor/question-editor.directive.ts

```diff
@@ -198,10 +201,12 @@ angular.module('oppia').directive('questionEditor', [
                 controller: 'ConfirmOrCancelModalController'
               }).result.then(function() {
                 updateQuestion(function() {
-                  recordedVoiceovers.markAllVoiceoversAsNeedingUpdate(
-                    contentId);
-                  writtenTranslations.markAllTranslationsAsNeedingUpdate(
-                    contentId);
+                  contentIds.forEach(contentId => {
```

Comment:
Shouldn't we be doing this only for the "unflagged voiceovers" rather than for all of them? (I'm not sure, and it might be worth adding some documentation to this function if there is additional context here that might help the reader.)